### ISSUE TITLE
GameGuru MAX - Bug - Animation Played Backward Plays Faster than Forward

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#1609]
I tried an experiment were I exported an FBX of a cube that goes up from 0-30 frames and not down animation. I imported this into MAX, name that animation channel as Open using the frames 0-30 and then created a second channel, named Close, and entered the first frame as 30 and the last as 0. Then I applied the Door behavior to the entity. It worked well. However, the Close animation played much faster than the Open animation. I don't think this should be happening since they are the same animation, the same number of frames, and there is nothing telling Close and Open to play at different speeds. I've tried this with a few different models just to make sure. I've even tried it with models that did have separate animations for Open and Close (but for this test had deleted the Close channel, added a new channel, named it Close, and use the Open channel frames, but backward).

**Comment by LeeBamberTGC** [comment]
@ArgentArts Please confirm this is fixed

**Comment by ArgentArts** [comment]
Will do. I'll look at it this afternoon. :)

**Comment by ArgentArts** [comment]
I just tested this and this is not working at all. I created a simple cube that moves to the right over 30 frames (0 - 29). When I created a new animation channel and set the frames start to frame 29 and the frame end to frame 0, the cube would not move. I even tried swapping the animation start and end frames for All and the cube would not move. If I swapped them back (so that Start was 0 and End was 29) then the cube moved accordingly.

So, previously, the animation used to play backwards, but much faster. Now, as of today's internal build, the animation does not play at all if reversed.
